.github/workflows/sentinel-build.yml

```diff
@@ -10,6 +10,13 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  validation:
+    name: "Validation"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gradle/actions/wrapper-validation@v4
```

Comment:
This workflow doesn't run on PRs, so I don't think it's super necessary here?

Reply:
Agree it's not super necessary, but my thought was that if the action fixes a bug or adds another check, this would catch something that made it to main.